lib/cdo/aws/cloud_formation.rb

```diff
@@ -323,30 +320,11 @@ def render_template(dry_run: false)
         erb_eval(template_string, filename)
       end
 
-      # Input string, output ERB-processed file contents in CloudFormation JSON-compatible syntax (using Fn::Join operator).
-      def source(str, filename, vars={})
-        local_vars = @@local_variables.dup
-        vars.each {|k, v| local_vars[k] = v}
-        lines = erb_eval(str, filename, local_vars).each_line.map do |line|
-          # Support special %{"Key": "Value"} syntax for inserting Intrinsic Functions into processed file contents.
-          line.split(/(%{.*})/).map do |x|
-            x =~ /%{.*}/ ? JSON.parse(x.gsub(/%({.*})/, '\1')) : x
-          end
-        end.flatten
-        {'Fn::Join' => ['', lines]}.to_json
-      end
```

Comment:
The special `%{"Key": "Value"}` syntax and splitting/joining template strings is no longer needed, with `Fn::Sub` we can now just use the special `${Ref}` syntax directly in the sourced string for making stack references, and don't need any of this post-processing code.